.github/workflows/scheduled-snapshots.yml

```diff
@@ -13,14 +13,14 @@ jobs:
     steps:
 
     - name: Configure AWS credentials
-      uses: aws-actions/configure-aws-credentials@v1
+      uses: aws-actions/configure-aws-credentials@v4
```

Comment:
Version 3 shortened the default session length from 6 hours to an hour, see https://github.com/aws-actions/configure-aws-credentials/blob/main/CHANGELOG.md#300-2023-08-21

I think that might be an issue for us, we should probably bump it back to 6 hours using the `role-duration-seconds` option.